templates/extern/owee.html.twig

```diff
@@ -181,12 +181,12 @@
 									Tijdens deze meeloopdag mag je een dagje meelopen met een student hier in Delft. Deze
 									dag zal volledig op jouw wensen en voorkeuren worden aangepast. Meer info is hier te vinden: {% endtrans %}
 							<a href="https://csrdelft.nl/meeloopdag">Informatie meeloopdag</a></h6>
-							<h6 class="open-avond">{% trans %}We hebben twee keer dit jaar een open avond waarbij je kan komen kijken hoe een avond in onze sociëteit eruit ziet. Je kan komen eten, meedoen met een praise uur en komen borrelen. {% endtrans %}</h6>
-
-							<p class="open-avond">{% trans %}De eerstvolgende open avond is donderdag 14 april 2022.{% endtrans %}</p>
+							<h6 class="open-avond">{% trans %}We hebben twee keer dit jaar een open avond waarbij je kan komen kijken hoe een avond in onze sociëteit eruit ziet. Je kan komen eten, meedoen met een worshipavond en komen borrelen. De eerstvolgende openavond is op 2 juni, de inschrijvingen daarvoor gaan 2 weken van tevoren open. {% endtrans %}</h6>
 
 							<div id="melding"></div>
-							<div class="velden" id="formulierVelden">
+
+{#						Alle drie de formulieren gebruiken dezelfde naam, email, telefoon en opmerking form! #}
```

Comment:
Iemand nog tips hoe dit efficienter kan? Ideaal gezien zou je de naam, email, telefoon en opmerking zichtbaar maken voor zowel 'lid-spreken' en 'lid-worden'. Dit lukte mij alleen niet, dus heb ik het er nu twee keer in staan.